ghcide/src/Development/IDE/GHC/Compat.hs

```diff
@@ -283,3 +283,12 @@ pattern ExposePackage s a mr <- DynFlags.ExposePackage s a _ mr
 #else
 pattern ExposePackage s a mr = DynFlags.ExposePackage s a mr
 #endif
+
```

Comment:
Seems like `splitLHsForAllTy` no longer exists since ghc 8.10 and there's now just [splitLHsForAllTyInvis](https://hackage.haskell.org/package/ghc-8.10.1/docs/GHC-Hs-Types.html#v:splitLHsForAllTyInvis).
Is this way of handling this situation ok?